### ISSUE TITLE
Include Python wheel releases for `factory_boy>=3.0.0`

### ISSUE THREAD

**Issue by jonathan-s** [FactoryBoy/factory_boy#780]
#### Description
Installing factory_boy > 3.0 gives me the following error

```
Running setup.py (path:/private/var/folders/t0/45yvxfds0gsgyvd3147j2sk00000gp/T/pip-build-uyeee7f3/factory-boy/setup.py) 
egg_info for package factory-boy produced metadata for project name unknown. 
Fix your #egg=factory-boy fragments.
```

Thus it's not installed as factory_boy, but as unknown.

**Comment by rbarrois** [comment]
I can't reproduce this issue locally, in a clean virtualenv.

Could you provide a complete sequence of commands that produces the issue you describe?

As far as I can tell, this is a problem with your environment:
```
/tmp % python -m virtualenv venv
Using base prefix '/usr/lib/python-exec/python3.7/../../..'
New python3.7 executable in /tmp/venv/bin/python
Also creating executable in /tmp/venv/bin/python3.7
Installing setuptools, pip, wheel...
done.

/tmp % source ./venv/bin/activate

(venv) /tmp % ./venv/bin/pip install factory_boy
Collecting factory_boy
  Downloading factory_boy-3.0.1.tar.gz (155 kB)
     |████████████████████████████████| 155 kB 12.9 MB/s 
Collecting Faker>=0.7.0
  Downloading Faker-4.1.2-py3-none-any.whl (1.0 MB)
     |████████████████████████████████| 1.0 MB 17.8 MB/s 
Collecting python-dateutil>=2.4
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 42.1 MB/s 
Collecting text-unidecode==1.3
  Downloading text_unidecode-1.3-py2.py3-none-any.whl (78 kB)
     |████████████████████████████████| 78 kB 76.6 MB/s 
Collecting six>=1.5
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Building wheels for collected packages: factory-boy
  Building wheel for factory-boy (setup.py) ... done
  Created wheel for factory-boy: filename=factory_boy-3.0.1-py3-none-any.whl size=35615 sha256=d4d1164fe6dcad7cb184a27f8cada8731219079b7510fa55effdc44e1a77188e
  Stored in directory: /tmp/pip-ephem-wheel-cache-tsj1xm_9/wheels/26/29/e6/ebf82f3421e634008b791258cb8aed2c6128627b141163db0c
Successfully built factory-boy
Installing collected packages: six, python-dateutil, text-unidecode, Faker, factory-boy
Successfully installed Faker-4.1.2 factory-boy-3.0.1 python-dateutil-2.8.1 six-1.15.0 text-unidecode-1.3

(venv) /tmp % ./venv/bin/pip freeze
factory-boy==3.0.1
Faker==4.1.2
python-dateutil==2.8.1
six==1.15.0
text-unidecode==1.3

**Comment by jonathan-s** [comment]
```
Elize:test jonathan$ python --version
Python 3.6.1
Elize:test jonathan$ python -m venv venv
Elize:test jonathan$ source venv/bin/activate
(venv) Elize:test jonathan$ pip install -U pip
Collecting pip
  Using cached https://files.pythonhosted.org/packages/4e/5f/528232275f6509b1fff703c9280e58951a81abe24640905de621c9f81839/pip-20.2.3-py2.py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 9.0.1
    Uninstalling pip-9.0.1:
      Successfully uninstalled pip-9.0.1
Successfully installed pip-20.2.3
(venv) Elize:test jonathan$ pip install factory_boy
Collecting factory_boy
  Using cached factory_boy-3.0.1.tar.gz (155 kB)
  WARNING: Generating metadata for package factory-boy produced metadata for project name unknown. Fix your #egg=factory-boy fragments.
Using legacy 'setup.py install' for unknown, since package 'wheel' is not installed.
Using legacy 'setup.py install' for unknown, since package 'wheel' is not installed.
Installing collected packages: unknown
    Running setup.py install for unknown ... done
Successfully installed unknown-0.0.0
(venv) Elize:test jonathan$ pip freeze
UNKNOWN==0.0.0
(venv) Elize:test jonathan$
```

**Comment by rbarrois** [comment]
Indeed; I still can't reproduce locally, even under Python 3.6:

```
/tmp % python3.6 -m venv venv

/tmp % . ./venv/bin/activate

(venv) /tmp % pip install --no-cache-dir --upgrade pip
Collecting pip
  Downloading https://files.pythonhosted.org/packages/4e/5f/528232275f6509b1fff703c9280e58951a81abe24640905de621c9f81839/pip-20.2.3-py2.py3-none-any.whl (1.5MB)
    100% |████████████████████████████████| 1.5MB 14.7MB/s 
Installing collected packages: pip
  Found existing installation: pip 18.1
    Uninstalling pip-18.1:
      Successfully uninstalled pip-18.1
Successfully installed pip-20.2.3

(venv) /tmp % pip install --no-cache-dir factory_boy
Collecting factory_boy
  Downloading factory_boy-3.0.1.tar.gz (155 kB)
     |████████████████████████████████| 155 kB 4.5 MB/s 
Collecting Faker>=0.7.0
  Downloading Faker-4.1.2-py3-none-any.whl (1.0 MB)
     |████████████████████████████████| 1.0 MB 20.2 MB/s 
Collecting text-unidecode==1.3
  Downloading text_unidecode-1.3-py2.py3-none-any.whl (78 kB)
     |████████████████████████████████| 78 kB 18.2 MB/s 
Collecting python-dateutil>=2.4
  Downloading python_dateutil-2.8.1-py2.py3-none-any.whl (227 kB)
     |████████████████████████████████| 227 kB 10.1 MB/s 
Collecting six>=1.5
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Using legacy 'setup.py install' for factory-boy, since package 'wheel' is not installed.
Installing collected packages: text-unidecode, six, python-dateutil, Faker, factory-boy
    Running setup.py install for factory-boy ... done
Successfully installed Faker-4.1.2 factory-boy-3.0.1 python-dateutil-2.8.1 six-1.15.0 text-unidecode-1.3

(venv) /tmp % pip freeze
factory-boy==3.0.1
Faker==4.1.2
python-dateutil==2.8.1
six==1.15.0
text-unidecode==1.3
```

Which OS are you using? How has your Python 3.6 been installed? If you try with `pip install factory-boy`, does it work properly?

**Comment by jonathan-s** [comment]
Using OSX 10.14.6, I've got python installed through pyenv. Does not work with `pip install factory-boy`. Happy to answer any other questions you might have. But if you feel like this might be only me and my odd computer also feel free to close the issue.

**Comment by morenol** [comment]
I also experienced this in the python:3.5-alpine and the python:3.8-alpine docker images. I also saw that problem in the python:3.8 docker image. Locally in my computer (ubuntu 20.04) that problem is not happening.

**Comment by rbarrois** [comment]
OK, now I've just noticed, for some reason we haven't been pushing wheels for the 3.0.1 / 3.0.0 releases!

**Comment by MRigal** [comment]
@rbarrois this really blocks some updating process! Please consider a quick release to address this issue!

Thanks in advance!

**Comment by MRigal** [comment]
I guess the behaviour is only reproducible with some (outdated?) versions of pip and setuptools

**Comment by rbarrois** [comment]
I've just released version 3.1.0 which comes with wheels, this should fix the issue you were encountering.

Please reopen the issue if it isn't working for you, or if you need a wheel for 3.0.0 / 3.0.1!
